internal/fileutils/fileutils.go

```diff
@@ -1017,3 +1018,38 @@ func ModTime(path string) (time.Time, error) {
 	}
 	return stat.ModTime(), nil
 }
+
+func CaseSensitivePath(path string) (string, error) {
+	var searchPath string
+	var err error
+	if runtime.GOOS != "windows" {
+		searchPath = caseSensitiveSearchPath(path)
+	} else {
+		volume := filepath.VolumeName(path)
+		remainder := strings.TrimLeft(path, volume)
+		searchPath = filepath.Join(volume, caseSensitiveSearchPath(remainder))
+	}
+
+	matches, err := filepath.Glob(searchPath)
+	if err != nil {
+		return "", errs.Wrap(err, "Failed to search for path")
+	}
+
+	if len(matches) == 0 {
+		return "", errs.New("Could not find path: %s", path)
+	}
+
+	return matches[0], nil
+}
+
+func caseSensitiveSearchPath(path string) string {
```

Comment:
Based on your explanation on Slack maybe we should call this `GlobPath()` or `UnixPath()` (latter not being technically correct, but much easier to interpret)?

Reply:
Thank you! I was having trouble thinking of names for this.